Missing Trait Implementations for `FlowBox` and Potential Conflicts

Hello, I'm enjoying the process of designing an app thus var with relm! However, I've run into an issue when looking for elements that might enable the option for more dynamic layouts like `FlowBox`. 

When trying to place items within FlowBox like this example codeblock:

```rust
view! {
   ...
            #[name(overview)]
            gtk::FlowBox::builder()
                .css_classes(vec!["overview"])
                .build()
            {
                set_orientation: gtk::Orientation::Horizontal,
                set_spacing: 10,

                #[name(title)]
                gtk::Label::builder()
                    .label(&self.0.entry.name)
                    .css_classes(vec!["title"])
                    .build(),
                #[name(comment)]
                gtk::Label::builder()
                    .label(&self.0.entry.comment.clone().unwrap_or_else(|| "".to_owned()))
                    .css_classes(vec!["comment"])
                    .visible(self.0.comments)
                    .build(),
            },
   ...
}
```

rust comes back with the following errors:

```
error[E0599]: the method `container_add` exists for struct `FlowBox`, but its trait bounds were not satisfied
...
   | |_doesn't satisfy `FlowBox: RelmContainerExt`
   |   doesn't satisfy `FlowBox: RelmSetChildExt`
   |
   = note: the following trait bounds were not satisfied:
           `FlowBox: RelmSetChildExt`
           which is required by `FlowBox: RelmContainerExt`
```

I'm assuming this is because `RelmContainerExt` and `RelmSetChildExt` are not implemented for `FlowBox`. With little understanding of the codebase, I tried to implement them myself but it seems like there are potential conflicts in doing so:
https://github.com/Relm4/Relm4/blob/main/relm4/src/extensions/remove.rs#L11
https://github.com/Relm4/Relm4/blob/main/relm4/src/extensions/remove.rs#L30

I'm not sure how to proceed. Is it possible to use a FlowBox in the way I want to here? Any feedback is appreciated. Thanks!

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -3,6 +3,7 @@
 ## Unreleased
 + components: Add `icon` to `IconButton`
 + core: Implement container traits to allow easier usage of libpanel widgets in the view macro
+* core: Support naturally appending any `Widget`s as children to `ListBox` and `FlowBox` in view macro.
 
 ### Added
 

--- a/relm4/src/extensions/container.rs
+++ b/relm4/src/extensions/container.rs
@@ -71,7 +71,7 @@ macro_rules! add_impl {
     };
 }
 
-append_impl!(gtk::Box, gtk::ListBox);
+append_impl!(gtk::Box, gtk::ListBox, gtk::FlowBox);
 add_child_impl!(gtk::InfoBar, gtk::Stack);
 
 #[cfg(feature = "libadwaita")]

--- a/relm4/src/extensions/mod.rs
+++ b/relm4/src/extensions/mod.rs
@@ -162,11 +162,13 @@ container_child_impl! {
     gtk::InfoBar,
     gtk::Button,
     gtk::ComboBox,
+    gtk::FlowBox,
     gtk::FlowBoxChild,
     gtk::Frame,
     gtk::Popover,
     gtk::Window,
     gtk::ApplicationWindow,
+    gtk::ListBox,
     gtk::ListBoxRow,
     gtk::ScrolledWindow,
     gtk::Dialog,
@@ -177,11 +179,6 @@ container_child_impl! {
     gtk::WindowHandle,
     gtk::Expander,
     gtk::AspectFrame
-}
-
-container_child_impl! {
-    gtk::ListBox: gtk::ListBoxRow,
-    gtk::FlowBox: gtk::FlowBoxChild
 }
 
 #[cfg(feature = "libadwaita")]

--- a/relm4/src/extensions/remove.rs
+++ b/relm4/src/extensions/remove.rs
@@ -23,22 +23,6 @@ where
     }
 }
 
-impl RelmRemoveExt for gtk::ListBox {
-    fn container_remove(&self, widget: &impl AsRef<Self::Child>) {
-        let row = widget.as_ref();
-        row.set_child(None::<&gtk::Widget>);
-        self.remove(row);
-    }
-}
-
-impl RelmRemoveExt for gtk::FlowBox {
-    fn container_remove(&self, widget: &impl AsRef<Self::Child>) {
-        let child = widget.as_ref();
-        child.set_child(None::<&gtk::Widget>);
-        self.remove(widget.as_ref());
-    }
-}
-
 #[cfg(feature = "libadwaita")]
 #[cfg_attr(docsrs, doc(cfg(feature = "libadwaita")))]
 impl RelmRemoveExt for adw::PreferencesGroup {
@@ -90,7 +74,9 @@ remove_impl!(
     gtk::Grid,
     gtk::ActionBar,
     gtk::Stack,
-    gtk::HeaderBar
+    gtk::HeaderBar,
+    gtk::ListBox,
+    gtk::FlowBox
 );
 remove_child_impl!(gtk::InfoBar);
 

